### DISCUSSION TITLE
I'd love to see the concept extended to a task app.

### DISCUSSION THREAD

**Discussion by gitgitdown** [cpvalente/ontime#1243] [Ideas]
For several years now, I've been looking for a task app that wouldn't be based on absolute days and hours, but would rather work on the logic of a workflow preset or a daily or weekly personal work routine that could be launched any day, at any time.

Play preset (Total time: 1h45): 
Ex: 
Task 1: Create a repository (5 min)
Subtask 1
Subtask 2
Task 2: Create a branch (10 min)
Subtask 1
Task 3: Make and commit changes (30 min)
Subtask 1
Task 4: Pick the kids up from school and remind myself with a geolocated reminder to buy chocolate (1 hour)

Lets say, I don't have time to do task 3, I can add it to another “preset”. This other preset will then add to its total duration, the time of the task I couldn't do.

Here it is, something like that, if you're interested in these ideas, for this or any other project.

**Comment by cpvalente** [comment]
Hi @gitgitdown, I apologise that we never got around to give you a reply.

I am not sure this is the correct forum to suggest a different line of work. From my side at least, there is no time to pursue extra side projects

I will close this ticket as it is unrelated to Ontime. Good luck and let me know if you ever make your app
